src/ets/scala/io/iohk/ethereum/ets/blockchain/BlockchainSuite.scala

```diff
@@ -0,0 +1,92 @@
+package io.iohk.ethereum.ets.blockchain
+
+import io.iohk.ethereum.ets.common.TestOptions
+import io.iohk.ethereum.ledger.BlockExecutionError
+import io.iohk.ethereum.network.p2p.messages.CommonMessages.NewBlock
+import io.iohk.ethereum.utils.Logger
+import org.scalatest._
+
+
+class BlockchainSuite extends FreeSpec with Matchers with Logger {
+
+  val unsupportedNetworks = Set("Byzantium","Constantinople", "EIP158", "EIP158ToByzantiumAt5", "HomesteadToDaoAt5")
+  val supportedNetworks = Set("EIP150", "Frontier", "FrontierToHomesteadAt5", "Homestead", "HomesteadToEIP150At5")
+
+  val ignoredTestNames = Set(
+    "bcForkUncle.json",
+    "ForkStressTest.json",
+    "GasLimitHigherThan2p63m1.json",
+    "CallContractFromNotBestBlock.json",
+    "ChainAtoChainB_blockorder1.json",
+    "ChainAtoChainBtoChainA.json",
+    "lotsOfBranches.json",
+    "lotsOfBranchesOverrideAtTheMiddle.json",
+    "lotsOfLeafs.json",
+    "sideChainWithMoreTransactions.json",
+    "uncleBlockAtBlock3afterBlock4.json",
+    "blockChainFrontierWithLargerTDvsHomesteadBlockchain.json",
+    "blockChainFrontierWithLargerTDvsHomesteadBlockchain2.json"
+  )
```

Comment:
Proposed changes:
- select individual tests (scenarios), not whole files (`Map[group-name, Set[scenario-name]`?)
- full group names rather than just file names (in case of duplicates)
- explicitly `cancel` these tests